db/migrations/schema-objects/20240103120853.do.rename-table-psychometricreport.sql

```diff
@@ -0,0 +1,7 @@
+IF EXISTS(SELECT *
+            FROM INFORMATION_SCHEMA.TABLES
+           WHERE TABLE_NAME = 'psychometricReport'
+             AND TABLE_SCHEMA = 'mtc_results')
+BEGIN
+    EXEC sp_rename 'mtc_results.psychometricReport', 'psychometricReport_2023'
```

Comment:
remember that these migrations won't get run on the 2023 production instance (i have literally just retargeted prod to a new 2024 instance).

So should we be dropping the table instead?  seems like the cleaner route...

Reply:
Happy to change to drop now that prod has been retargeted to a new DB.